artifacts/camara-cloudevents/event-subscription-template.yaml

```diff
@@ -949,11 +1004,23 @@ components:
           schema:
             $ref: "#/components/schemas/ErrorInfo"
           examples:
-            PermissionDenied:
+            MULTIEVENT_SUBSCRIPTION_NOT_SUPPORTED:
               value:
                 status: 422
                 code: "MULTIEVENT_SUBSCRIPTION_NOT_SUPPORTED"
                 message: "Multi event types subscription not managed"
+            GENERIC_422_DEVICE_IDENTIFIERS_MISMATCH:
```

Comment:
I understand this is for APIs that manage device concept to have exceptions for the case of indicating a non-applicable device (e.g. networkAccessIdentifier) or providing several identifiers that are inconsistent. OK to me

cc @fernandopradocabrillo, @jlurien